doc/pins/p4rt_app_hld.md

```diff
@@ -164,6 +165,30 @@ P4RT application introduces new tables that are written to APPL_DB for the table
 
 The P4RT application code will have unit & component tests that together will give >80% code coverage.
 
+## Configuring P4RT Application
+
+The P4RT application is configured at the start-up by reading the P4RT configuration from the CONFIG_DB. If no valid config exists in CONFIG_DB, it uses the default values. The configuration can be added to the CONFIG_DB by being manually added to the config_db.json file. The P4RT container will need to be restarted if the configuration is changed.
+
+Below is an example of adding P4RT configuration to config_db.json. It initializes the P4RT application with the default configuration (which can be omitted). The user can modify this block based on their environment settings. 
+
+```
+"P4RT": {
+  "certs": {
+    "server_crt": "/keys/server_cert.lnk",
+    "server_key": "/keys/server_key.lnk",
+    "ca_crt": "/keys/ca_cert.lnk",
+    "cert_crl_dir": "/keys/crl"
+  },
+  "p4rt_app": {
+    "port": "9559",
+    "use_genetlink": "false",
+    "use_port_ids": "false",
+    "save_forwarding_config_file" : "/etc/sonic/p4rt_forwarding_config.pb.txt",
+    "p4rt_unix_socket" : "/sock/p4rt.sock",
```

Comment:
nit: I would also remove the `p4rt_unix_socket` argument.

While it is configurable, it has a very specific use case that I don't think most people staring in PINS would be interested in.